src/main.rs

```diff
@@ -32,32 +32,50 @@ struct Post {
     contents: String,
     url: String,
     published: String,
+    release: bool,
 }
 
 #[derive(Debug, PartialEq, Serialize, Deserialize)]
 struct YamlHeader {
     title: String,
     author: String,
+    #[serde(default = "default_release")]
```

Comment:
The default value for bools is false.

Reply:
I was getting errors leaving default as:
```suggestion
    #[serde(default = "false")]
```
or

```suggestion
    #[serde(default = false)]
```
So implemented it as a function. Is this ok?

Reply:
You just want `#[serde(default)]`.